tests/Fixtures/TestBundle/Serializer/Denormalizer/DiscrAbstractDummyDenormalizer.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Again, the problem resides in the fact that we cannot identify the type of an element within the collection without providing that information, that's why the supportsDenormalization method checks for a "@type" property in the provided data ...
Is it a "won't fix" issue ?